packages/api-reference/src/components/Sidebar.vue

```diff
@@ -179,17 +179,48 @@ const activeSidebarItemId = computed(() => {
 
   return flattenedItems.find((item) => isVisible(item.id))?.id ?? null
 })
+
+// This offset determines how far down the sidebar the items scroll
+const SCROLL_OFFSET = -160
+const scrollerEl = ref<HTMLElement | null>(null)
+
+// Watch for the active item changing so we can scroll the sidebar
+watch(activeSidebarItemId, (activeId) => {
+  const el = document.getElementById(`active-sidebar-element-id-${activeId!}`)
```

Comment:
I think we should do refs here

https://vuejs.org/guide/essentials/template-refs.html#refs-inside-v-for

Reply:
> It should be noted that the ref array does not guarantee the same order as the source array.
So then how do you access the ref after? Loop through the array?

Reply:
Alright had to do some vue learning, they made some changes from vue 2. Check it out now